js/main.js

```diff
@@ -0,0 +1,53 @@
+//constants
+const timeout_between_levels = 2000; //in milliseconds
+const frogs_color = ['AliceBlue','AntiqueWhite','Aqua','Aquamarine','Azure','Beige','Bisque','Black','BlanchedAlmond','Blue','BlueViolet','Brown','BurlyWood','Chartreuse','Chocolate','Coral','Cornsilk','Crimson','Cyan','DarkBlue','DarkCyan','DarkGoldenRod','DarkGray','DarkGrey','DarkGreen','DarkKhaki','DarkMagenta','DarkOliveGreen','DarkOrange','DarkOrchid','DarkRed','DarkSalmon','DarkSlateBlue','DarkSlateGray','DarkTurquoise','DarkViolet','DeepPink','DeepSkyBlue','DimGray','DimGrey','DodgerBlue','FireBrick','FloralWhite','ForestGreen','Fuchsia','Gainsboro','GhostWhite','Gold','GoldenRod','Green','GreenYellow','HoneyDew','HotPink','IndianRed','Indigo','Ivory','Khaki','Lavender','LavenderBlush','LawnGreen','LemonChiffon','LightBlue','LightCoral','LightCyan','LightGoldenRodYellow','LightGray','LightGreen','LightPink','LightSalmon','LightSkyBlue','LightSteelBlue','LightYellow','Lime','LimeGreen','Linen','Magenta','Maroon','MediumAquaMarine','MediumBlue','MediumOrchid','MediumPurple','MediumSlateBlue','MediumSpringGreen','MediumTurquoise','MediumVioletRed','MidnightBlue','MintCream','MistyRose','Moccasin','NavajoWhite','Navy','OldLace','Olive','Orange','OrangeRed','Orchid','PaleGoldenRod','PaleGreen','PaleTurquoise','PaleVioletRed','PapayaWhip','PeachPuff','Peru','Pink','Plum','PowderBlue','Purple','RebeccaPurple','Red','RosyBrown','RoyalBlue','SaddleBrown','Salmon','SandyBrown','SeaShell','Sienna','Silver','SkyBlue','SlateBlue','SlateGray','Snow','SpringGreen','Tan','Teal','Thistle','Tomato','Turquoise','Violet','Wheat','White','WhiteSmoke','Yellow','YellowGreen'];
+const frogs_div_text = 'Frogs : ';
+const levels_div_text = 'Level : ';
+const catch_the_frogs_text = 'Catch the frogs!';
+const timeout_text = 'Time : ';
+const star_button_text = 'Start Game';
+const default_start_button_warning_color = '#f16612';
+const default_start_button_danger_color = 'red';
+
+//one time declarations
+let level;
+let frogs_counter;
+let timeout_level;
+let start_button;
```

Comment:
there is no reason for you to declare these before the code starts, it just adds more lines of code. You give the button an actual value on line 39 so you should just declare it there. It will make your code a lot easier to read.

Reply:
I made these to make it easier for other developers to make changes,
for example like: you change only these lines and u will get new results

instead of digging into code